.github/workflows/integrationtests.yml

```diff
@@ -13,7 +13,7 @@ jobs:
   # Build
   Build:
     name: Build
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
```

Comment:
Ubuntu 24 would require to manually install Mono, which will result in a slower build